doc/source/data/loading-data.rst

```diff
@@ -790,35 +714,62 @@ Call :func:`~ray.data.read_sql` to read data from a database that provides a
                 "SELECT year, COUNT(*) FROM movie GROUP BY year", create_connection
             )
 
+Reading NoSQL databases
+~~~~~~~~~~~~~~~~~~~~~~~
 
-.. _data_custom_datasource:
+Ray Data reads from MongoDB.
 
-------------------
-Custom Datasources
-------------------
+.. tab-set::
 
-Datasets can read and write in parallel to :ref:`custom datasources <data_source_api>` defined in Python.
-Once you have implemented `YourCustomDataSource`, you can use it like any other source in Ray Data:
+    .. tab-item:: MongoDB
 
-.. code-block:: python
+        To read data from MongoDB, call :func:`~ray.data.read_mongo` and specify the
+        the source URI, database, and collection. You also need to specify a pipeline to
+        run against the collection.
 
-    # Read from a custom datasource.
-    ds = ray.data.read_datasource(YourCustomDatasource(), **read_args)
+        .. testcode::
+            :skipif: True
 
-    # Write to a custom datasource.
-    ds.write_datasource(YourCustomDatasource(), **write_args)
+            import ray
 
-For more details, read :ref:`Implementing a Custom Datasource <custom_datasources>`.
+            # Read a local MongoDB.
+            ds = ray.data.read_mongo(
+                uri="mongodb://localhost:27017",
+                database="my_db",
+                collection="my_collection",
+                pipeline=[{"$match": {"col": {"$gte": 0, "$lt": 10}}}, {"$sort": "sort_col"}],
+            )
+
+            # Reading a remote MongoDB is the same.
+            ds = ray.data.read_mongo(
+                uri="mongodb://username:password@mongodb0.example.com:27017/?authSource=admin",
+                database="my_db",
+                collection="my_collection",
+                pipeline=[{"$match": {"col": {"$gte": 0, "$lt": 10}}}, {"$sort": "sort_col"}],
+            )
+
+            # Write back to MongoDB.
+            ds.write_mongo(
+                MongoDatasource(),
+                uri="mongodb://username:password@mongodb0.example.com:27017/?authSource=admin",
+                database="my_db",
+                collection="my_collection",
+            )
+
+Loading unsupported data
+========================
```

Comment:
Just call this Unsupported data sources?

Reply:
"Loading other data sources" sounds a bit better to me. Because they actually "supported", but just need a custom datasources.